### ISSUE TITLE
Add PGID and PUID support for all-in-one docker container

### ISSUE THREAD

**Issue by p2bauer** [HaveAGitGat/Tdarr#153]
**Is your feature request related to a problem? Please describe.**
Tdarr is working great in my setup so far (synology, using docker) - but the only sticking point is that without support for PGID and PUID environment variables, it runs under the default user...and every transcoded file gets put back with the wrong permissions.

**Describe the solution you'd like**
Implement support for PGID and PUID env vars, like most linuxserver docker containers (see https://docs.linuxserver.io/general/understanding-puid-and-pgid).

**Describe alternatives you've considered**
 I know there is also the `--user` docker flag, but synology doesn't yet support that....and I would guess other docker installs may not yet either.

**Comment by p2bauer** [comment]
I've just noticed after joining discord and searching - there were some docker compose examples using these variables.  Are they also in the main image in docker hub & I'm just having issues on my end?

**Comment by marshalleq** [comment]
There were a few problems with this, but it does appear to be working for me now.  I just checked a recently processed file to confirm it.  This is what I have set in unraid to give you the idea.
<img width="626" alt="Screen Shot 2020-02-06 at 08 52 49" src="https://user-images.githubusercontent.com/13308996/73877745-1619d800-48be-11ea-9bdf-afadf1655c5c.png">

<img width="952" alt="Screen Shot 2020-02-06 at 08 50 59" src="https://user-images.githubusercontent.com/13308996/73877693-fa163680-48bd-11ea-8234-fa11da127878.png">
